app/assets/stylesheets/modules/brief-view.scss

```diff
@@ -1,20 +1,13 @@
 .brief-document {
   @include document;
-  margin-bottom: 0px;
+  margin-bottom: -1px;
   padding-left: 20px;
 
   h3.index_title {
     @include h3-index-title-small-font;
     word-wrap: break-word;
   }
 
-  .preview {
```

Comment:
the `preview` class is not nested under `brief-document` in the DOM, so these rules are never applied.